src/config/2020.json

```diff
@@ -0,0 +1,225 @@
+          "title": "Third Parties",
+          "slug": "third-parties"
+        },
+        {
+          "part": "I",
+          "chapter": "6",
+          "title": "Fonts",
+          "slug": "fonts"
+        }
+      ]
+    },
+    {
+      "part": "II. User Experience",
+      "part_number": "2",
+      "chapters": [
+        {
+          "part": "II",
+          "chapter": "7",
+          "title": "Performance",
+          "slug": "performance"
+        },
+        {
+          "part": "II",
+          "chapter": "8",
+          "title": "Security",
+          "slug": "security"
+        },
+        {
+          "part": "II",
+          "chapter": "9",
+          "title": "Accessibility",
+          "slug": "accessibility"
+        },
+        {
+          "part": "II",
+          "chapter": "10",
+          "title": "SEO",
+          "slug": "seo"
+        },
+        {
+          "part": "II",
+          "chapter": "11",
+          "title": "PWA",
+          "slug": "pwa"
+        },
+        {
+          "part": "II",
+          "chapter": "12",
+          "title": "Mobile Web",
+          "slug": "mobile-web"
+        }
+      ]
+    },
+    {
+      "part": "III. Content Publishing",
+      "part_number": "3",
+      "chapters": [
+        {
+          "part": "III",
+          "chapter": "13",
+          "title": "Ecommerce",
+          "slug": "ecommerce"
+        },
+        {
+          "part": "III",
+          "chapter": "14",
+          "title": "CMS",
+          "slug": "cms"
+        }
+      ]
+    },
+    {
+      "part": "IV. Content Distribution",
+      "part_number": "4",
+      "chapters": [
+        {
+          "part": "IV",
+          "chapter": "15",
+          "title": "Compression",
+          "slug": "compression"
+        },
+        {
+          "part": "IV",
+          "chapter": "16",
+          "title": "Caching",
+          "slug": "caching"
+        },
+        {
+          "part": "IV",
+          "chapter": "17",
+          "title": "CDN",
+          "slug": "cdn"
+        },
+        {
+          "part": "IV",
+          "chapter": "18",
+          "title": "Page Weight",
+          "slug": "page-weight"
+        },
+        {
+          "part": "IV",
+          "chapter": "19",
+          "title": "Resource Hints",
+          "slug": "resource-hints"
+        },
+        {
+          "part": "IV",
+          "chapter": "20",
+          "title": "HTTP/2",
+          "slug": "http2"
+        }
+      ]
+    }
+  ],
+  "teams": {
+    "analysts": {
+      "name": "Analysts"
+    },
+    "authors": {
+      "name": "Authors"
+    },
+    "brainstormers": {
+      "name": "Brainstormers"
+    },
+    "designers": {
+      "name": "Designers"
+    },
+    "developers": {
+      "name": "Developers"
+    },
+    "editors": {
+      "name": "Editors"
+    },
+    "reviewers": {
+      "name": "Reviewers"
+    },
+    "translators": {
+      "name": "Translators"
+    }
+  },
+  "contributors": {
```

Comment:
We could leave `contributors` empty and add to it when we have a Contributors page available.

Reply:
Similar to above, I see no harm in adding it now. I also think it would make it easier for contributors to add themselves as we go along. Some people may do some development, then bore and drop out - like happened last year. It was a bit of an effort last time to get people added to this after the fact and we even forgot one or two.